apps/common/helpers.py

```diff
@@ -0,0 +1,28 @@
+from random import choices
+from string import ascii_letters, digits
+import stripe
+
+from config.settings import env
+
+stripe.api_key = env('STRIPE_SECRET_TEST_API_KEY')
+
+
+def decimal_to_int_stripe(money):
+    """
+    Used to format decimal price for charge.
+    """
+    return int(money * 100)
+
+
+def generate_code() -> str:
```

Comment:
hai totusi sa facem un cod mai mic de 6 cifre, si poti schimba un pic formatarea la functie, sa intins prea mult in jos